src/xtopcom/xgasfee/src/xgas_tx_operator.cpp

```diff
@@ -142,12 +142,12 @@ bool xtop_gas_tx_operator::is_one_stage_tx() const {
 }
 
 evm_common::u256 xtop_gas_tx_operator::balance_to_tgas(const evm_common::u256 balance) {
-    xassert(XGET_ONCHAIN_GOVERNANCE_PARAMETER(tx_deposit_gas_exchange_ratio) > 0);
-    return balance / XGET_ONCHAIN_GOVERNANCE_PARAMETER(tx_deposit_gas_exchange_ratio);
+    xassert(g_tx_deposit_fee > 0);
```

Comment:
why g_tx_deposit_fee?

Reply:
to prevent gas adjustment by tcc account.